testing.py

```diff
@@ -20,3 +26,17 @@
 
 testSuite.runSelectedTests()
 testSuite.report()
+
+parseXML()
+rpc_connect("eagle33")
+
+setTriggerSource(testSuite.glib,options.gtx,1)
+configureLocalT1(testSuite.glib, options.gtx, 1, 0, 40, 250, 0, options.debug)
+print 'Starting OH T1 controller'
+startLocalT1(testSuite.glib, options.gtx, options.debug)
+reg=getNode("GEM_AMC.OH.OH%i.T1Controller.TOGGLE"%options.gtx)
+print 'RPC register real_address ',hex(reg.real_address)
+print 'RPC register address ',hex(reg.address)
+#writeReg(reg,0x1)
```

Comment:
This looks suspiciously like RPC related stuff, which should not be part of this PR